mobile-widgets/qml/DivePlannerSetup.qml

```diff
@@ -10,6 +10,7 @@ import org.kde.kirigami 2.4 as Kirigami
 Kirigami.ScrollablePage {
 	title: qsTr("Dive planner setup")
 
+	property string textLength: (PrefUnits.length === "meters") ? qsTr("m/min") : qsTr("ft/min")
```

Comment:
I an not in love with the name of that property. Wouldn't this be `speedUnit` or something along those line?

Reply:
changed